lib/protobuf/wire/varint.ex

```diff
@@ -163,7 +163,7 @@ defmodule Protobuf.Wire.Varint do
 
   defp def_decoder_success_clauses(name, args, body) do
     for {pattern, expression} <- @varints do
-      quote do
+      quote location: :keep do
```

Comment:
Why are we changing this?

Reply:
I changed that to find where the problem was - the message was just saying this:
![warning](https://user-images.githubusercontent.com/86847575/148216755-5f0b4474-8440-4272-9980-4fd8a352561d.jpg)
 
 Which just points to [this line](https://github.com/elixir-protobuf/protobuf/blob/main/lib/protobuf/decoder.ex#L121). I'm happy to revert that if you prefer?

Reply:
Cool if this change fixes the warning, Im good with it! Does it? 😄